portfolio.css

```diff
@@ -408,7 +430,189 @@ form > ul {
   border-radius: 8px;
 }
 
-.form-btn:hover {
-  background-color: #172b4d;
-  color: #fff;
+::placeholder {
+  color: #b3bac5;
+}
+
+.desktop-post {
+  display: none;
+}
+
+.menu-list {
+  display: none;
+}
+
+/* =========MEDIAL QUERRY============== */
+
+@media only screen and (min-width: 800px) {
+  .mheadline {
+    width: 100%;
+    height: 90vh;
+    background-image: url(images/big-header-svg.svg);
+    background-size: contain;
+    background-position: center;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .portfolio-class {
+    width: 100%;
+  }
+
+  .desktop-about-section {
+    width: 50%;
+  }
+
+  .desktop-about-section > h3 {
+    font-weight: 400;
+    font-size: 24px;
+    line-height: 60px;
+  }
+
+  .desktop-about-section > p {
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  .icon-stack-container {
+    display: none;
+  }
+
+  .menu-list {
+    display: inline;
+  }
+
+  .menu-list > li {
+    list-style-type: none;
+    margin: 0;
+    padding: 15px;
+    display: inline;
+  }
+
+  nav {
+    padding: 15px 143px 100px 143px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .projects {
+    grid-template-columns: 1fr 3.3fr 1fr;
+  }
+
+  .card {
+    width: 100%;
+    flex-direction: row;
+  }
+
+  .mobile-work-img {
+    display: none;
+  }
+
+  .desktop-wrok-img {
+    display: flex;
+  }
+
+  .desktop-work-detail {
+    padding: 35px;
+  }
+
+  .desktop-work-detail > p {
+    font-size: 16px;
+  }
+
+  .desktop-work-detail > h3 {
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .card > .desktop-work-detail > .project_details > li {
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  .reverse-work,
+  .reverse-work-2 {
+    flex-direction: row-reverse;
+  }
+
+  .preview_button {
+    margin-top: 25px;
+  }
+
+  .hide {
+    display: none;
+  }
+
+  .desktop-post {
+    display: flex;
+  }
+
+  .about {
+    width: 100%;
+    display: grid;
+    place-content: center;
+    grid-template-columns: 1fr 2fr;
+    grid-template-rows: 1fr;
```

Comment:
- [ ] As mentioned by the [previous reviewer](https://github.com/Alabi12/Mobile_First/pull/7/files#r984587348), your design looks great, well-done. But one aim of this project is allowing you to practice attention to detail which is 100% expected in real work situations. The about section text `Hello I'm a software developer!.......` has a higher with than what you currently have. I think if you use `grid-template-columns: 2fr 2fr;` it would be better.  Also, your language section should take all width for this section.

See the image below, The part selected red, should be placed at the part selected blue. And the fadded border-bottom line should span through the yellow line. Overall, look at this section on the Figma design again and try to make it look almost the same as the design. 
**_Please note that in the image below, I added `grid-template-columns: 2fr 2fr;` that is why the about text width look wider._**

![image](https://user-images.githubusercontent.com/84156714/193327688-9fff3302-78a2-458a-ab8e-ba46060d2518.png)